engine/src/test/java/io/camunda/zeebe/engine/util/TestStreams.java

```diff
@@ -322,10 +322,10 @@ public StreamProcessor getStreamProcessor(final String streamName) {
             () -> new NoSuchElementException("No stream processor found with name: " + streamName));
   }
 
-  public long writeBatch(final String logName, final RecordToWrite[] recordToWrites) {
+  public LogStreamBatchWriter batchWriter(
+      final String logName, final RecordToWrite[] recordToWrites) {
```

Comment:
🔧  I think this could have a better name as well because the current name looks like a retrieval method on a record, but actually, it builds and sets up the logstream batch writer with the provided records to write. I think the name should reflect that.